### PR TITLE
DSC19-146: Add QuestionnaireResponse to DHIS2 mapping and flow

### PR DESCRIPTION
**Pull request by de-laz**

### CODE REVIEW

File: docker/importer/volume/surveillance-mediator/case-outcome/endpoint-resource-resolver-dhis.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
+{
+  "name": "Covid19 Case Outcome DHIS2 Orchestrator",
+  "description": "Endpoint for mapping the Covid19 case outcome into DHIS format and then forwarding it to DHIS2",
+  "endpoint": {
+    "pattern": "/resource-resolver-dhis"
```
Comment (MattyJ007) [inline docker/importer/volume/surveillance-mediator/case-outcome/endpoint-resource-resolver-dhis.json]: The path doesn't match?

**Review by MattyJ007 — CHANGES_REQUESTED** [review]
Where's the Kubernetes?

File: docker/importer/volume/surveillance-mediator/case-outcome/endpoint-resource-resolver-dhis.json (hunk 1 above)
Comment (de-laz) [reply docker/importer/volume/surveillance-mediator/case-outcome/endpoint-resource-resolver-dhis.json]: @MattyJ007 Where is the required path documented?

Comment (MattyJ007) [reply docker/importer/volume/surveillance-mediator/case-outcome/endpoint-resource-resolver-dhis.json]: You added the path to the endpoint-handler file

File: kubernetes/importer/volume/surveillance-mediator/case-outcome/endpoint-resource-resolver-dhis.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    "incidentDate": "$now()"
+  },
+  "requests": {
+    "response": [
+      {
+        "primary": true,
+        "id": "dhis-file-queue",
+        "config": {
+          "method": "POST",
+          "url": "http://file-queue:4002/populator",
```
Comment (bradsawadye) [inline kubernetes/importer/volume/surveillance-mediator/case-outcome/endpoint-resource-resolver-dhis.json]:
```suggestion
          "url": "http://covid19-surveillance-file-queue-service:4002/populator",
```

**Pull request merged**